packages/store/test/test-heap-classes.js

```diff
@@ -150,6 +150,7 @@ test('sloppy option', t => {
     () =>
       makeExo(
         'greeter',
+        // @ts-expect-error missing guard
         M.interface('greeter', emptyBehavior, { sloppy: false }),
```

Comment:
Is this statically complaining for the same reason it is dynamically complaining? That there is a mismatch between the `interfaceGuard` argument and the `methods` argument? That's awesome!

Reply:
yeah!